pkg/sdkserver/sdk.go

```diff
@@ -66,5 +67,13 @@ func convert(gs *agonesv1.GameServer) *sdk.GameServer {
 		result.Status.Ports = append(result.Status.Ports, grpcPort)
 	}
 
+	if runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		result.Status.Players = &sdk.GameServer_Status_PlayerStatus{}
```

Comment:
Is the idea to always include an empty Players block even if gs.Status.Players is nil? Why not do

```
if gs.Status.Players != nil {
  result.Status.Players = &sdk.GameServer_Status_PlayerStatus{
    Count: gs.Status.Players.Count,
    Capacity: gs.Status.Players.Capacity,
  }
}
```

Reply:
That's a good question. My original thought was to have less opportunities for panics on nil, but I think your way is better. I'll make the change :+1: